contracts/StrategySushiswapPair.sol

```diff
@@ -134,6 +136,7 @@ contract StrategySushiswapPair is BaseStrategy {
      */
     function prepareReturn(uint256 _debtOutstanding) internal override returns (uint256 _profit, uint256 _loss, uint256 _debtPayment) {
         if (_debtOutstanding > 0) {
+            //this will crash is debtOutstaning is more than amount you have
```

Comment:
I do not believe it is technically possible because it is a lossless strategy, but I will see if I can add a nice fix for it anyways